packages/astro/src/events/session.ts

```diff
@@ -107,7 +107,11 @@ function createAnonymousConfigInfo(userConfig: AstroUserConfig) {
 	};
 	// Measure string literal/enum configuration values
 	configInfo.build.format = measureStringLiteral(userConfig.build?.format);
-	configInfo.markdown.syntaxHighlight = measureStringLiteral(userConfig.markdown?.syntaxHighlight);
+	const syntaxHighlightType =
+		typeof userConfig.markdown?.syntaxHighlight === 'object'
+			? userConfig.markdown?.syntaxHighlight.type
+			: userConfig.markdown?.syntaxHighlight;
+	configInfo.markdown.syntaxHighlight = measureStringLiteral(syntaxHighlightType);
```

Comment:
This only measures for the syntax highlight type. Is that ok?

Reply:
Yeah I believe it's OK!